cranelift/codegen/src/isa/x64/inst/mod.rs

```diff
@@ -57,7 +57,7 @@ pub struct ReturnCallInfo<T> {
 fn inst_size_test() {
     // This test will help with unintentionally growing the size
     // of the Inst enum.
-    assert_eq!(56, std::mem::size_of::<Inst>());
+    assert_eq!(48, std::mem::size_of::<Inst>());
```

Comment:
Nice! This is presumably because a reg that we carried as a (always the same value) `u32` previously now becomes a type parameter? Good to see reductions in any case.